sdk/containerregistry/azure-containerregistry/samples/sample_upload_download_manifest.py

```diff
@@ -0,0 +1,80 @@
+# coding: utf-8
+
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+"""
+FILE: sample_upload_download_manifest.py
+
+DESCRIPTION:
+    This sample demonstrates uploading and downloading an OCI manifest to a repository.
+
+USAGE:
+    python sample_upload_download_manifest.py
+
+    Set the environment variables with your own values before running the sample:
+    1) CONTAINERREGISTRY_ENDPOINT - The URL of you Container Registry account
+
+    This sample assumes your registry has a repository "library/hello-world".
+"""
+import os
+import json
+from io import BytesIO
+from dotenv import find_dotenv, load_dotenv
+from azure.containerregistry import ContainerRegistryClient
+from azure.containerregistry._generated.models import Annotations, Descriptor, OCIManifest
+from sample_utilities import load_registry, get_authority, get_audience, get_credential
+
+
+class UploadDownloadManifest(object):
+    def __init__(self):
+        load_dotenv(find_dotenv())
+        self.endpoint = os.environ.get("CONTAINERREGISTRY_ENDPOINT")
+        self.authority = get_authority(self.endpoint)
+        self.audience = get_audience(self.authority)
+        self.credential = get_credential(self.authority, exclude_environment_credential=True)
+        load_registry()
+
+    def upload_download_manifest(self):
+        repository_name = "library/hello-world"
+        layer = BytesIO("Sample layer".encode())
```

Comment:
```suggestion
        layer = BytesIO(b"Sample layer")
```

Reply:
What does the "layer" data represent? How would I normally build this for a manifest? I'm not sure this sample is very indicative of customer usage.